packages/ui-shell/README.md

```diff
@@ -0,0 +1,54 @@
+# @okta/ui-shell
+
+## Getting Started
+
+Install the package and peer dependencies:
+
+```sh
+yarn add @okta/ui-shell
+```
+
+Include fonts:
+
+```html
+<link rel="preconnect" href="https://fonts.googleapis.com" />
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+<link
+  href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600&display=swap"
+  rel="stylesheet"
+/>
```

Comment:
That's a great question! Yes, I can include these fonts in the Shadow DOM. Also, I think have to because I'm not sure if Shadow DOM includes global document fonts.

I wonder why no one's had issues running Odyssey in Shadow DOM without those font inclusions though.